assets/scss/_search.scss

```
@@ -122,39 +122,44 @@
 }
 
 .popover-body {
-  background-color:#2e2e31;
+  background-color: #2e2e31;
 }
 
 // Global Navigation Bar
-.td-navbar{
-  min-height:6rem;
+.td-navbar {
+  min-height: 6rem;
   background-color: $body-bg !important;
-  .navbar-brand  {
+
+  @media (max-width: 991.98px) {
```

Do we need to put `.td-navbar-nav-scroll` inside a breakpoint?
```suggestion
  .td-navbar-nav-scroll {
      max-width: fit-content;
    }
```